packages/@aws-cdk/aws-elasticloadbalancingv2-actions/README.md

```diff
@@ -0,0 +1,21 @@
+# Actions for AWS Elastic Load Balancing V2
+<!--BEGIN STABILITY BANNER-->
+---
+
+![cdk-constructs: Experimental](https://img.shields.io/badge/cdk--constructs-experimental-important.svg?style=for-the-badge)
+
+> The APIs of higher level constructs in this module are experimental and under active development. They are subject to non-backward compatible changes or removal in any future version. These are not subject to the [Semantic Versioning](https://semver.org/) model and breaking changes will be announced in the release notes. This means that while you may use them, you may need to update your source code when upgrading to a newer version of this package.
+
+---
+<!--END STABILITY BANNER-->
+
+This package contains integration actions for ELBv2. See the README of the `@aws-cdk/aws-elasticloadbalancingv2` library.
+
+## Cognito
+
+To use a Cognito UserPool for authentication, use `AuthenticateCognitoAction`. For details on the
+setup's requirements, read [Prepare to use Amazon Cognito](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/listener-authenticate-users.html#cognito-requirements).
+
+Here's an example:
+
+[Example of using AuthenticateCognitoAction](test/integ.cognito.lit.ts)
```

Comment:
Does this work on our documentation page? 
Would be nice to put a small code snippet inline, along with a link to the full example.

Reply:
I guess. That's a feature request to jsii at the moment.

Reply:
But we can still add an inline example manually.

Reply:
I've tried this example and it seems not to work for me while trying to use this with `ApplicationLoadBalancedFargateService`